extensions/industry/healthcare/fhir/fieldgroups/patient.schema.json

```diff
@@ -0,0 +1,241 @@
+                  "$ref":"https://ns.adobe.com/xdm/industry/healthcare/fhir/datatypes/humanname"
+               }
+            },
+            "xdm:telecom":{
+               "title":"Contact Details",
+               "description":"A contact detail (e.g. a telephone number or an email address) by which the individual may be contacted.",
+               "type":"array",
+               "items":{
+                  "$ref":"https://ns.adobe.com/xdm/industry/healthcare/fhir/datatypes/contactpoint"
+               }
+            },
+            "xdm:gender":{
+               "title":"Gender",
+               "description":"male | female | other | unknown.",
+               "type":"string",
+               "enum":[
+                  "male",
+                  "female",
+                  "other",
+                  "unknown"
+               ],
+               "meta:enum":{
+                  "male":"Male",
+                  "female":"Female",
+                  "other":"Other",
+                  "unknown":"Unknown"
+               }
+            },
+            "xdm:birthDate":{
+               "title":"Birth Date",
+               "type":"string",
+               "format":"date",
+               "description":"The date of birth for the individual"
+            },
+            "xdm:deceasedBoolean":{
+               "title":"Deceased Indicator",
+               "type":"boolean",
+               "description":"Indicates if the individual is deceased or not."
+            },
+            "xdm:deceasedDateTime":{
+               "title":"Deceased Date Time",
+               "type":"string",
+               "format":"date-time",
+               "description":"Date Time if the patient is deceased."
+            },
+            "xdm:address":{
+               "title":"Address",
+               "description":"Patient's Address.",
+               "type":"array",
+               "items":{
+                  "$ref":"https://ns.adobe.com/xdm/industry/healthcare/fhir/datatypes/address"
+               }
+            },
+            "xdm:maritalStatus":{
+               "title":"Marital Status",
+               "description":"Martitial status codes",
+               "$ref":"https://ns.adobe.com/xdm/industry/healthcare/fhir/datatypes/codeableconcept"
+            },
+            "xdm:multipleBirthBoolean":{
+               "title":"Is Part Of multipleBirth",
+               "type":"boolean",
+               "description":"Boolean flag indicating if the patient is part of a multiple birth"
+            },
+            "xdm:multipleBirthInteger":{
+               "title":"Birth Number",
+               "type":"integer",
+               "description":"Birth number in the sequence."
+            },
+            "xdm:photo":{
+               "title":"Patient Photo",
+               "description":"Pateint's photogragh",
+               "type":"array",
+               "items":{
+                  "$ref":"#/definitions/image-or-link-or-array-of-image-or-link"
```

Comment:
This reference seems wrong

Reply:
removed the field. need to find a new reference data type